mab2rec/pipeline.py

```diff
@@ -620,7 +620,8 @@ def _validate_bench(recommenders, metrics, train_data, test_data, cv):
                                   RankingRecoMetrics.Precision,
                                   RankingRecoMetrics.Recall,
                                   RankingRecoMetrics.NDCG,
-                                  RankingRecoMetrics.MAP)),
+                                  RankingRecoMetrics.MAP,
+                                  DiversityRecoMetrics.InterListDiversity)),
```

Comment:
Any reason why we can't also add IntraListDiversity here? From what I recall this should also run fine.

Reply:
I don't have a test case for it rn, that's why I didn't add it. Other than that I agree.

Reply:
while at it, it might be worth adding a test case and making this complete?